examples/features/src/bunnymark/mod.rs

```diff
@@ -21,12 +21,12 @@ struct Globals {
 }
 
 #[repr(C, align(256))]
-#[derive(Clone, Copy, Zeroable)]
+#[derive(Clone, Copy, Pod, Zeroable)]
 struct Bunny {
     position: [f32; 2],
     velocity: [f32; 2],
     color: u32,
-    _pad: u32,
+    _pad: [u32; (256 - 20) / 4],
```

Comment:
question: Is this a bugfix? 👀

Reply:
Kinda? It was formally UB before as we were casting padding bytes to part of a u8 array. Bytemuck cares about this a lot, so would not compile the derive until the padding was fixed.